openlxp_authentication/serializers.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+from .models import SAMLConfiguration
+
+
+class SAMLConfigurationSerializer(serializers.ModelSerializer):
+    """Serializes the SAMLConfiguration Model"""
+
+    class Meta:
+        model = SAMLConfiguration
+
+        fields = ['name', 'endpoint']
```

Comment:
Probably add a new line here, Do we have flake8 and isort check set for this package. It could help format things

Reply:
I have just formatted and sorted the imports for each file.  Let me know if you want any specific changes